packages/slate-react/src/components/android/use-android-input-manager.ts

```diff
@@ -17,7 +17,7 @@ export function useAndroidInputManager(node: RefObject<HTMLElement>) {
   const editor = useSlateStatic()
   const [inputManager] = useState(() => new AndroidInputManager(editor))
   const { receivedUserInput, onUserInput } = useTrackUserInput()
-  const timeoutId = useRef<NodeJS.Timeout | null>(null)
+  const timeoutId = useRef<ReturnType<typeof setTimeout> | null>(null)
```

Comment:
Please verify this is intentional? Might be better to raise this as a separate PR.

Reply:
I was getting a `Cannot find namespace 'NodeJS'` error when running the build with yarn 3. It's trying to resolve `NodeJS` to a package it seems and can't find it. After some googling this seemed a good way to set the type for a `setTimeout` as the `NodeJS.Timeout` type is used in node environments but this is browser code.

Reply:
Makes sense, thanks! It still might be better in a separate PR since it's not really part of yarn 3, up to you.

Reply:
Sounds good to me, made a separate PR here https://github.com/ianstormtaylor/slate/pull/4419